src/client/browser/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "hammerhead/use-hh-promise": 0
```

Comment:
JFYI, it uses `pinkie` since it's creation, there was no error just because linter was turned off.